spec/models/exit_type_spec.rb

```diff
@@ -13,5 +13,24 @@
     it_behaves_like OrganizationScope
 
     it { is_expected.to validate_presence_of(:name) }
+
+    context "with an associated mortality_events" do
+      subject(:exit_type) { create(:exit_type) }
+
+      before do
+        create(:mortality_event, exit_type: exit_type)
+      end
+
+      it "cannot be destroyed" do
+        expect { exit_type.destroy }
+          .not_to change(MeasurementType, :count)
```

Comment:
Just one minor change. I think this spec should check that the ExitType count has not changed.